core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -3018,6 +3020,223 @@ class KafkaApisTest {
       any())
   }
 
+  @Test
```

Comment:
Are we relying on the existing tests for the old group coordinator and the kafka apis changes there?

Reply:
Yes.